### ISSUE TITLE
Limit max font size (and page width)

### ISSUE THREAD

**Issue by waldyrious** [waldyrious/downstyler#47]
My personal experience is that often the font size is too big for comfortable reading, when using large screens, and end up using the browser zoom feature to reduce the font size (and consequently page width) to a comfortable size -- usually 80%. It would be nice if the style itself capped those sizes to a reasonable default. Or perhaps the growth rate could be reduced.

**Comment by waldyrious** [comment]
With the new [min(), max() and clamp()](https://stackoverflow.com/a/61085200/266309), not only this is easier to do, but even the current implementation (from #9) becomes simpler :)

Specifically, the current implementation:

```css
html { font-size: 16px; }
@media (min-width: 600px) { html { font-size: calc(13px + 0.5vw) } }
```

could become:
```css
html { font-size: max(16px, calc(13px + 0.5vw)); }
```

...and if we want to introduce a maximum:
```css
html { font-size: clamp(16px, calc(13px + 0.5vw), 20px); }
```

**Comment by waldyrious** [comment]
Fixed in #54.
